### PR TITLE
Fix combined deploy scripts

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

- Hot fix to deploy scripts. Dev deploy from main failed: https://github.com/CDCgov/prime-simplereport/actions/runs/596466355

## Changes Proposed

- access env var correctly

## Additional Information

- Ran the deploy to test: https://github.com/CDCgov/prime-simplereport/actions/runs/596498219
  - Frontend: https://test.simplereport.gov/app/static/commit.txt
  - Backend: Failed due to the database not being configured correctly 

- Ran the deploy to demo: https://github.com/CDCgov/prime-simplereport/runs/1971478533?check_suite_focus=true
  - Frontend: https://demo.simplereport.gov/app/static/commit.txt
  - Backend: https://demo.simplereport.gov/api/actuator/info

- Testing new/renamed Github actions is a bit tricky. In hindsight I think the approach should have been commit no-op action with the desired name to `main`. Add action steps, test on branch and remove old actions

### CODE REVIEW

File: .github/workflows/deployDemo.yml [hunk 1]
```diff
@@ -32,10 +32,10 @@ jobs:
           ./build_and_push.sh
       - uses: hashicorp/setup-terraform@v1
       - name: Terraform Init
-        working-directory: ${{ github.workspace }}/ops/${{ DEPLOY_ENV }}
+        working-directory: ${{ github.workspace }}/ops/${{ secrets.DEPLOY_ENV }}
```
Comment (benwarfield-usds) [inline .github/workflows/deployDemo.yml:35]:
```suggestion
        working-directory: ${{ github.workspace }}/ops/${{ env.DEPLOY_ENV }}
```

**Review by nickscialli-usds — APPROVED** [review]
Looks good. Seems like there's no reason to set all these other env variables if we're just still accessing the `secrets` object in the rest of the steps.

```
ARM_CLIENT_ID: ${{ secrets.TERRAFORM_ARM_CLIENT_ID }}
ARM_CLIENT_SECRET: ${{ secrets.TERRAFORM_ARM_CLIENT_SECRET }}
ARM_SUBSCRIPTION_ID: ${{ secrets.TERRAFORM_ARM_SUBSCRIPTION_ID }}
ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
ACR_ADMIN_USERNAME: ${{ secrets.ACR_ADMIN_USERNAME }}
ACR_ADMIN_PASWORD: ${{ secrets.ACR_ADMIN_PASWORD }}
ACR_REPO_URL: ${{ secrets.ACR_REPO_URL }}
```

**Comment by timothybest-usds** [comment]
It appears that the test api is missing `pgcrypto`
```
2021-02-24T15:37:48.011686962Z Caused by: liquibase.exception.DatabaseException: ERROR: function gen_random_uuid() does not exist
2021-02-24T15:37:48.011693662Z   Hint: No function matches the given name and argument types. You might need to add explicit type casts.
2021-02-24T15:37:48.011697562Z   Position: 125 [Failed SQL: (0) INSERT INTO public.api_user
2021-02-24T15:37:48.011701262Z (internal_id, created_at, updated_at, is_deleted, last_seen, login_email, last_name)
2021-02-24T15:37:48.011704962Z VALUES (
2021-02-24T15:37:48.011708562Z   gen_random_uuid(), now(), now(), false, null,
2021-02-24T15:37:48.011712262Z   'db-migrations@simplereport.gov', 'Database Migration Process'
2021-02-24T15:37:48.011715962Z )]
```

**Comment by nickscialli-usds** [comment]
@TimBest that's the error I was getting locally that I fixed by resetting the DB with the readme steps

**Comment by timothybest-usds** [comment]
merging to fix deploy scripts on main and then going to run `CREATE EXTENSION pgcrypto;` on the test env db

**Pull request merged**